src/main/java/org/apache/commons/csv/CSVParser.java

```diff
@@ -799,7 +806,10 @@ CSVRecord nextRecord() throws IOException {
 
     /**
      * Returns a sequential {@code Stream} with this collection as its source.
-     *
+     * <p>
+     * If the parser is closed, the stream will not produce any more values.
+     * See the comments in {@link iterator()}.
```

Comment:
@pedro-w For future reference: `{@link iterator()}` is NOT legal Javadoc syntax, use `{@link #iterator()}`